### DISCUSSION TITLE
Does Opentelemetry-Collector have any FIPS or FedRamp compatibility?

### DISCUSSION THREAD

**Discussion by Jugis** [open-telemetry/opentelemetry-collector#4212] [Q&A]
Hey folks, 

We would like to use OpenTelemetry on private cloud instalments and we have a certain FIPS and FedRamp compatibility criterias to meet. Does the OpenTelemetry-Collector have any documentation on wether it is compatible? Identified gaps maybe or any kind of information available on this topic?

Thanks in advance!

**Comment by bogdandrutu** [comment]
@Jugis I have no experience with FIPS or FedRamp, but happy to hear what we can do to help.

**Comment by ycombinator** [comment]
I just created #13925 which might be of interest.
